specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/InstancePoolOperations.json

```diff
@@ -0,0 +1,268 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/instancePools/{instancePoolName}/operations": {
```

Comment:
>operations [](http://example.com/codeflow?start=129&length=10)

doesnt this have a put method? How an operation is added/cancelled?

Reply:
There is no PUT for operations, they are created when PUT or PATCH is executed on parent resource (InstancePool) so we only need these (GET) methods for listing operations and monitoring their status.